packages/hardhat-foundry/src/getRemappings.ts

```diff
@@ -0,0 +1,33 @@
+import { runCmdSync } from "./runCmd";
+
+let remappings: { [from: string]: string };
+
+export function getRemappings() {
+  // Return remappings if they were already loaded
+  if (remappings !== undefined) {
+    return remappings;
+  }
+
+  // Get remappings from foundry
+  const remappingsTxt = runCmdSync("forge remappings");
+
+  remappings = {};
+  const remappingLines = remappingsTxt.split("\n");
```

Comment:
I'm not super sure, but I think this should be:

```suggestion
  const remappingLines = remappingsTxt.split(/\r|\n|\r\n/);
```

Reply:
Good catch, I moved the `\r\n` to the beginning of the regex to match before just `\r`